src/main/scala/fi/aalto/cs/codeInspection/AaltoInspectionBundle.java

```diff
@@ -0,0 +1,23 @@
+package fi.aalto.cs.codeInspection;
+
+import com.intellij.DynamicBundle;
+import org.jetbrains.annotations.Nls;
+import org.jetbrains.annotations.NonNls;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.PropertyKey;
+
+public final class AaltoInspectionBundle extends DynamicBundle {
```

Comment:
I would like to see code formatting ([example](https://github.com/Aalto-LeTech/SMILe/blob/main/.scalafmt.conf)) and [scaladocs](https://docs.scala-lang.org/style/scaladoc.html). The Scala files should follow the Scala 3 style (i.e. braceless). The html files could be formatted too

Reply:
Does this file have to be Java

Reply:
Added the Scalafmt and formatted the Scala files to the Scala 3 style. Will need to look at the html files separately, I remember the code blocks getting messed up by the formatting for some reason.